notCollected.php

```diff
@@ -14,14 +14,15 @@
 <body>
 <h1>Uncaught Pokemon</h1>
 <a href="index.php">Back to Collection</a>
-
+<div style="display: grid; grid-template-columns: 1fr 1fr 1fr; gap: 0px 0px;">
```

Comment:
Plz put me in a stylesheet! And also apply this to the collection page as well!

Reply:
U could make me the happiest trainer in the world if you also added a media query to drop the columns down to one on a small screen size, then I won't make you write any more CSS for the rest of the week 😉